frontend/src/scenes/feature-flags/FeatureFlagSnippets.tsx

```diff
@@ -163,6 +163,132 @@ export function APISnippet(): JSX.Element {
     )
 }
 
+export function JSMultivariateSnippet({ flagKey }: { flagKey: string }): JSX.Element {
+    return (
+        <CodeSnippet language={Language.JavaScript} wrap>
+            {`if (posthog.getFeatureFlag('${flagKey ?? ''}') === 'example-variant') {
```

Comment:
instead of example-variant, can this be the variant they choose?

Reply:
also `onFeatureFlags` thingy is needed here!

Reply:
how do you know which is the variant they choose? if they set up multiple variants, do you want me to just pick a random one?

Reply:
nah, like we do on the experiments screen:

<img width="1256" alt="image" src="https://user-images.githubusercontent.com/7115141/225061743-5d936b8d-ad05-4dc8-b954-a32c8766630e.png">

Reply:
Makes sense to reuse these components

Reply:
we'll see, in the meantime can you help think of a good TLDR for bootstrapping and local eval?

Reply:
Fair